core/src/main/java/com/linecorp/armeria/client/limit/AbstractConcurrencyLimitingClient.java

```diff
@@ -54,17 +57,14 @@
 public abstract class AbstractConcurrencyLimitingClient<I extends Request, O extends Response>
         extends SimpleDecoratingClient<I, O> {
 
-    private static final long DEFAULT_TIMEOUT_MILLIS = 10000L;
-
-    private final int maxConcurrency;
-    private final long timeoutMillis;
-    private final AtomicInteger numActiveRequests = new AtomicInteger();
     private final Queue<PendingTask> pendingRequests = new ConcurrentLinkedQueue<>();
+    private final ConcurrencyLimit concurrencyLimit;
+    private final AtomicInteger numActiveRequests;
 
     /**
      * Creates a new instance that decorates the specified {@code delegate} to limit the concurrent number of
-     * active requests to {@code maxConcurrency}, with the default timeout of {@value #DEFAULT_TIMEOUT_MILLIS}
-     * milliseconds.
+     * active requests to {@code maxConcurrency}, with the default timeout of
+     * {@link ConcurrencyLimitBuilder.DEFAULT_TIMEOUT_MILLIS} milliseconds.
```

Comment:
Hide `ConcurrencyLimitBuilder.DEFAULT_TIMEOUT_MILLIS` from public API?
```suggestion
     * {@value ConcurrencyLimitBuilder.DEFAULT_TIMEOUT_MILLIS} milliseconds.
```